js/posts/postManagement/updatePost/updateConnection.mjs

```diff
@@ -0,0 +1,17 @@
+import {
+  api_Base_Url,
+  api_Update_Post_EndPoint,
+} from "../../../api/api_Url_Endpoints.mjs";
+import { updatePost } from "./index.mjs";
+
+export function updateUserPost(container, postData) {
+  container
+    .querySelector("form#updatePostForm")
+    .addEventListener("submit", (e) => {
+      e.preventDefault();
+      updatePost(`${api_Base_Url}${api_Update_Post_EndPoint}${postData.id}`);
+      setTimeout(() => {
+        location.reload();
```

Comment:
You should use async/await not timeout

Reply:
Tried going async/await and the post wouldnt update at all.